### DISCUSSION TITLE
How to add CA cert for schema registry https connection

### DISCUSSION THREAD

**Discussion by stephaneuh** [provectus/kafka-ui#2468] [Q&A] [answered]
Hello, our Schema Registry is exposing an SSL certificate from our internal Certificate Authority.
Where should we put this root CA in the kafka-ui filesystem, so kafka-ui can connect to schema registry?

Could it be:
/etc/ssl/certs/java/cacerts

Thanks,

**Comment by Haarolean** [comment, marked as answer]
Hey, SSL for SR is not yet implemented. Feel free to upvote #930 and subscribe to updates :)
